Common/Gfx/Canvas.h

```diff
@@ -74,8 +75,13 @@ class Canvas
 	void DrawMaskedBitmap(Gdiplus::Bitmap* bitmap, Gdiplus::Bitmap* maskBitmap, const Gdiplus::Rect& dstRect,
 		const Gdiplus::Rect& srcRect, const Gdiplus::Rect& srcRect2);
 
-	void FillRectangle(Gdiplus::Rect& rect, const Gdiplus::SolidBrush& brush);
+	//Needed to create shapes in meters, change it if you want. If you find a simpler way to expose this with templates etc
+	static Microsoft::WRL::ComPtr<ID2D1RectangleGeometry> CreateRectangle(D2D1_RECT_F rectangle);
```

Comment:
`const D2D1_RECT_F&`, please. Can you also move this to the top (below `Finalize`) near the other static functions.